infra-sofa-boot-starter/src/main/java/com/alipay/sofa/infra/listener/SofaBootstrapRunListener.java

```diff
@@ -28,10 +29,7 @@
 import org.springframework.boot.context.event.ApplicationEnvironmentPreparedEvent;
 import org.springframework.context.ApplicationListener;
 import org.springframework.core.Ordered;
-import org.springframework.core.env.ConfigurableEnvironment;
-import org.springframework.core.env.MapPropertySource;
-import org.springframework.core.env.PropertySource;
-import org.springframework.core.env.StandardEnvironment;
+import org.springframework.core.env.*;
```

Comment:
Avoid use import * here. You can change your IDEA "imports" setting to 999...

Reply:
ok, fixed